ui/src/pages/AllStories/index.tsx

```diff
@@ -552,6 +623,13 @@ export const AllStories: React.FC = () => {
           <StyledEmptyMessage> No pending changes! </StyledEmptyMessage>
         )}
       </AllStoriesTabs>
+      <StoryDrawer
+        story={clickedStory}
+        onClose={() => setClickedStory(undefined)}
+        onClickEditStory={() => {
+          console.log("TODO: Route to edit page");
```

Comment:
Did you try adding this route with a Redirect?

Reply:
That one's to route to edit stories which needs to get merged in from your PR?